### PR TITLE
{devel}[dummy/] Spark v2.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2498.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9fcaaad3b936ecffeb562612e2ff240a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2140.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f12ab7a4ed26beef667d8ab2552a212a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1fedca906952b1bf94f4148a3942a7e0 for a full test report.

File: easybuild/easyconfigs/s/Spark/Spark-2.2.0-Hadoop-2.6.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'Tarball'
+
+name = 'Spark'
+version = '2.2.0'
+versionsuffix = '-Hadoop-2.6'
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Spark/Spark-2.2.0-Hadoop-2.6.eb]: @boegel Is it worth adding `Java` to the `versionsuffix`?

Comment (boegel) [inline easybuild/easyconfigs/s/Spark/Spark-2.2.0-Hadoop-2.6.eb]: I guess it is, done

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**